odoo/modules/registry.py

```diff
@@ -295,10 +298,14 @@ def setup_models(self, cr, partial=False):
         for model in models:
             model._setup_complete()
 
+    def post_init(self, func, *args, **kwargs):
+        """ Register a function to call at the end of :meth:`~.init_models`. """
+        self._post_init_queue.append(partial(func, *args, **kwargs))
```

Comment:
Out of 9 calls, 8 need arguments. For instance,
```python
self.pool.post_init(record.modified, list(params))
```
would be replaced by
```python
self.pool.post_init(functools.partial(record.modified, list(params)))
```
In fact the first API makes the calls to `post_init` easier to read, IMHO.